test/jdk/java/awt/Frame/AwtFramePackTest.java

```diff
@@ -67,32 +84,35 @@ public static void main(String[] args) throws AWTException {
             mb.add(m);
             frame.setMenuBar(mb);
 
+            frame.setResizable(isResizable);
             frame.pack();
             frame.setVisible(true);
 
-            robot.delay(500);
             robot.waitForIdle();
+            robot.delay(500);
 
             Dimension actualFrameSize = frame.getSize();
             Dimension expectedFrameSize = frame.getPreferredSize();
 
             if (!actualFrameSize.equals(expectedFrameSize)) {
-                System.out.println("Expected frame size: "+ expectedFrameSize);
-                System.out.println("Actual frame size: "+ actualFrameSize);
-                saveScreenCapture();
-                throw new RuntimeException("Expected and Actual frame size" +
-                        " are different. frame.pack() does not work!!");
+                System.out.println("Expected frame size: " + expectedFrameSize);
+                System.out.println("Actual frame size: " + actualFrameSize);
+                saveScreenCapture(frameType + ".png");
+                errorLog.append(frameType + ": Expected and Actual frame size" +
```

Comment:
`frameType` is used only inside this failure block, so it's declaration can be moved here before it's used for the first time.